draft-ietf-quic-multipath.md

```diff
@@ -1274,6 +1274,12 @@ are not congestion controlled (see {{Section 7 of QUIC-RECOVERY}}),
 senders should carefully consider the load induced
 by these packets, especially if the capacity is unknown on that path.
 
+The specification of scheduling for concurrent multipath and related the
+congestion control algorithms and re-ordering methods for use in the general
```

Comment:
I appreciate this text comes from almost directly from MP-DCCP. However, 

1. "concurrent mulitpath" doesn't seem to be a term or phrase this draft uses anywhere
2. I think there's mistake or bad grammar in "concurrent multipath and related the congestion control algorithms"
3. don't understand the "re-ordering methods" part - is that even relevant to this design?

Reply:
I agree with the above comments (especially point 3, as there should not be any packet reordering over a given path, but rather some "STREAM data" reordering, which is handled by design in (single-path) QUIC). More generally, do we need to add more than "Further scheduling considerations are implementation specific and are out of scope of this document."?

Reply:
This document is not about IP though. So IP packet reordering methods seem totally put of scope of discussion. There's literally nothing I can do with that statement as an implementer of this extension